gallery/src/components/NxDrawer/NxDrawerWithNxFormExample.tsx

```diff
@@ -0,0 +1,224 @@
+            <NxDrawer.Header.Title>An Example of A Drawer with Form</NxDrawer.Header.Title>
+            <NxDrawer.Header.Subtitle>Header Subtitle</NxDrawer.Header.Subtitle>
+            <NxDrawer.Header.Description>Header Description</NxDrawer.Header.Description>
+          </NxDrawer.Header>
+
+          <NxForm className="nx-form"
+                  onSubmit={() => setShowDrawer(false)}
+                  onCancel={() => setShowDrawer(false)}>
+            <NxDrawer.Content>
+              <NxFormGroup label="A Field to Fill in" isRequired>
+                <NxStatefulTextInput aria-required={true} validator={validator}/>
+              </NxFormGroup>
+              <NxFormGroup label="Country" sublabel="Pick your favorite from the list">
+                <select className="nx-form-select" value={val} onChange={onChange}>
+                  <option value="">Pick a Country</option>
+                  <option value="USA">USA</option>
+                  <option value="GER">Canada</option>
+                  <option value="CAN">Germany</option>
+                  <option value="COL">Colombia</option>
+                </select>
+              </NxFormGroup>
+              <NxFieldset label="Colors" isRequired>
+                <NxCheckbox onChange={toggleRed} isChecked={isRed}>Red</NxCheckbox>
+                <NxCheckbox onChange={toggleBlue} isChecked={isBlue}>Blue</NxCheckbox>
+                <NxCheckbox onChange={toggleGreen} isChecked={isGreen}>Green</NxCheckbox>
+              </NxFieldset>
+            </NxDrawer.Content>
+          </NxForm>
+        </NxDrawer>
+      )}
+
+      {showDrawerOverflowing && (
+      <NxDrawer id="nx-drawer-with-nx-form-overflowing"
+                onCancel={() => setShowDrawerOverflowing(false)}
+                aria-label="Drawer with Overflowing Form">
+        <NxDrawer.Header>
+          <NxDrawer.Header.Title>Drawer With A Long Form</NxDrawer.Header.Title>
+          <NxDrawer.Header.Subtitle>Header Subtitle</NxDrawer.Header.Subtitle>
+          <NxDrawer.Header.Description>Header Description</NxDrawer.Header.Description>
+        </NxDrawer.Header>
+
+        <NxForm className="nx-form"
+                onSubmit={() => setShowDrawerOverflowing(false)}
+                onCancel={() => setShowDrawerOverflowing(false)}>
+          <NxDrawer.Content>
+            <NxFieldset label="Enable features" isRequired>
+              <NxToggle inputId="subscribe-check" onChange={toggleWarp} isChecked={isWarpOn}>
+                Enable Warp Drive
+              </NxToggle>
+              <NxToggle inputId="no-label-check" onChange={toggleKraken} isChecked={isKrakenOut}>
+                Release the Kraken!
+              </NxToggle>
+              <NxToggle inputId="children-check" onChange={toggleShapes} isChecked={isShapes}>
+                Allow shapes
+              </NxToggle>
+            </NxFieldset>
+            <NxFormGroup label="Username" isRequired>
+              <NxTextInput aria-required={true}
```

Comment:
`isRequired` on the `NxFormGroup` takes care of this
```suggestion
              <NxTextInput
```